pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1267,4 +1267,16 @@ self: super: {
   # Haddock failure: https://github.com/haskell/haddock/issues/979
   esqueleto = dontHaddock (dontCheck super.esqueleto);
 
+  # Get hpack-convert to compile with the versions of its dependencies in nixpkgs.
+  # The tests are still broken
+  hpack-convert = dontCheck (overrideSrc super.hpack-convert {
+    version = "1.0.2";
```

Comment:
As far as I can see, 1.0.2 is not released? https://github.com/yamadapc/hpack-convert/releases